pkg/expr/expr.go

```diff
@@ -25,6 +25,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	lrucache "github.com/hashicorp/golang-lru"
```

Comment:
just call this import `lru`. this matches the package definition and avoids stutter like `*lrucache.Cache`